### DISCUSSION TITLE
Details highlight bar missing on current version?

### DISCUSSION THREAD

**Discussion by bradthornborrow** [jarun/nnn#1649] [Q&A]
Hey there, just recently discovered nnn and discovered an idiosyncrasy between the new and old version.  When in details view, the old version on my Debian systems (3.5.1) highlights the entire file and details selected.  On the current version in my Ubuntu system (4.6.1 I recall), it only highlights the filename, and the details are not highlighted.  

Is this a bug with the version shipped with Ubuntu, or is this a feature of v4.x?  I prefer the old version which highlights the entire row and details.  Is there a build or environment setting I can change to get that back?  Any help would be appreciated!  Thanks.

**Comment by N-R-K** [comment]
> Is this a bug with the version shipped with Ubuntu, or is this a feature of v4.x?

It's probably intended because it works the same way on my gentoo system.

> Is there a build or environment setting I can change to get that back?

I don't think so. But if you git bisect the commit down then you can maybe revert that commit locally.

**Reply by bradthornborrow** [reply]
Thanks, did some experimentation, and it seems like version 3.7.1 was the culprit, and found this in the release notes.
- show only file name in reverse in detail mode

Now just rebuild 4.8 with this reversion, although would be nice if the devs added this as a build or environment option.  Perhaps I'll put in a PR if I can figure out the specifics needed....

**Reply by bradthornborrow** [reply]
Surprisingly, I figured it out. Three minor code changes in the printent function restore the full highlight bar. Unfortunately, this conflicts with the "highlight timestamp for files 5 minutes young", but changing that to use A_UNDERLINE avoids the conflict, and actually looks better (in my humble opinion).  

That said, as this is a fairly minor feature that it appears @jarun removed back in 3.7, do you think there's any chance this functionality being restored if I submitted a PR? Being new to this project, not sure how to proceed, other than fork and maintain my own version.

**Reply by N-R-K** [reply]
> do you think there's any chance this functionality being restored if I submitted a PR?

That entirely up to jarun, so I'd wait for his reply.

**Comment by jarun** [comment]
It was removed intentionally. No plans to bring it back.
